code/parachain/runtime/picasso/src/assets.rs

```diff
@@ -0,0 +1,68 @@
+use super::*;
+
+pub type BalanceIdentifier = [u8; 8];
+
+type MaxLocks = ConstU32<64>;
+
+pub struct DustRemover;
```

Comment:
@dzmitry-lahoda do you agree with such dust handler? simply specifying Treasury pallet doesnt work because it doesnt implement required trait

Reply:
problem that you may double drop

Reply:
likeley instead of Peek use Offset. But tbh, just use () - read the code - it may just drop, drop calls relevant code. like in Acala.

so i think this code may not be needed.

Reply:
reverted, please check it does not double mints. not it will burn.